spec/models/metadata_spec.rb

```diff
@@ -123,12 +139,61 @@
       expect(created_location.osm_id).to eq(200)
     end
 
+    # This tests the cache by saving the same raw_value twice.
+    it "should return correct response if called twice with the same raw_value" do
+      # This needs to be different from all other locations in this file, so as not to trigger the cache.
```

Comment:
what guarantees do we have that the cache will always be on during unit tests?

Reply:
Hey @gregdingle. It looks like we already testing the cache elsewhere so I assumed it would be on. Also, it looks like in config/environments/test.rb, we set the cache_store to Redis.

I remember you worked on the cache though, so please let me know if you still have concerns!

Reply:
Ok that's good to know. Caching is something I would expect could be turned on or off in tests depending on the application.